pkg/ddc/base/runtime_helper.go

```diff
@@ -33,6 +33,8 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var fuseDeviceResourceName string
```

Comment:
You can add `getFuseDeviceResourceName()` to wrap fuseDeviceResourceName. The purpose is to avoid  handling variable directly.

Reply:
Done